### ISSUE TITLE
New colony expeditions

### ISSUE THREAD

**Issue by tom-pratt** [jsettlers/settlers-remake#591]
Just an idea for a feature that's not in the original game. I saw that @rpolzer is working on the ships (nice!)

I always thought that starting a new colony was so slow and annoying in settlers 3 that no one ever does it. I think we should add in the idea of expeditions like in Settlers 2.

A trade ship would take something like, 5 carriers, 2 soldiers, essential tools, wood and stone. When it reaches the destination it automatically takes some land and starts building a guard house.

No need to bother with pioneers and a ferry ship. Could also do a donkey expedition which would be the same sort of thing from a marketplace.

**Comment by rpolzer** [comment]
I used to see making a new colony as a challenge and prefer the complicated way in Settlers 3 compared with the easy way in Settler 2 - but that is just my personal preference.

**Comment by Tom-Pratt** [comment]
Yes true, it should definitely be a game option so you can still play with the original format.

I think it could be a nice option though, especially when playing against human players, trying to start a second colony with the original system feels like a bit of a tactical disadvantage because they take so long to establish. This option could make colonies a more integral part of the game

**Comment by FinnPeak** [comment]
Hello,

I think this sounds like a good idea. You're correct, the original method is so painful that it never gets done in multiplayer games. Combined if game will include bigger maps than original, this might create some nice, probably still quite minor in importance, tactics.
